packages/expo-cellular/src/ExpoCellular.web.ts

```diff
@@ -0,0 +1,34 @@
+import { CellularGeneration } from './Cellular.types';
+
+export default {
+  get allowsVoip(): null {
+    return null;
+  },
+  get carrier(): null {
+    return null;
+  },
+  get isoCountryCode(): null {
+    return null;
+  },
+  get mobileCountryCode(): null {
+    return null;
+  },
+  get mobileNetworkCode(): null {
+    return null;
+  },
+  async getCellularGenerationAsync(): Promise<CellularGeneration> {
+    let connection =
+      navigator['connection'] || navigator['mozConnection'] || navigator['webkitConnection'];
```

Comment:
```suggestion
      navigator['connection'] || navigator['mozConnection'] || navigator['webkitConnection'] || {};
```

otherwise `connection.effectiveType` could null-crash, right? 🤔